### PR TITLE
Remove expiration blocks

### PR DESCRIPTION
**Pull request by netcriptus**

### CODE REVIEW

**Review by weilbith — CHANGES_REQUESTED** [review]
Thanks for taking care of this! 👍🏾 

There is on more occurrence in a comment at `source/raiden/SecretRegistry.sol:15`.

File: raiden_contracts/tests/unit/test_unit_signatures.py [hunk 1]
```diff
@@ -20,51 +20,51 @@ def test_recover_address_from_withdraw_message(
     deposit_A = 5
     deposit_B = 7
     withdraw_A = 3
-    expiration_block = 492889
+    withdrawable_until = 492889 * 15
```
[the remark this answers is not in the document]
Comment (netcriptus) [reply raiden_contracts/tests/unit/test_unit_signatures.py:23]: Our best approximation for future blocks is still to consider 15 seconds/block

[the remark this answers is not in the document]
Comment (weilbith) [reply raiden_contracts/tests/unit/test_unit_signatures.py:23]: Can maybe have a call for this?

[the remark this answers is not in the document]
Comment (weilbith) [reply raiden_contracts/tests/unit/test_unit_signatures.py]: In our call we found out that this number is completely agnostic for the test and can be even `1`. I like to express this simpleness of the value to keep the test clean. But the value before was very specific, though its seems like it was random. As compromise with Fernando we agreed to leave it on the old value, but definitely remove the confusing `* 15`.

**Review by weilbith — APPROVED** [review]
✔️

**Pull request merged**